docs/conf.py

```diff
@@ -25,12 +25,17 @@
 # be accessible, and the documentation will not build correctly.
 # See sphinx_astropy.conf for which values are set there.
 
-from datetime import datetime
 import os
 import sys
 
 import configparser
-from pkg_resources import get_distribution
+from datetime import datetime
```

Comment:
Not sure why you moved `datetime`. Shouldn't it be grouped with stdlib imports?

Reply:
I don't remember, I guess I was using isort but not sure why `datetime` moved here since indeed it's from stdlib.

Reply:
And btw isort is broken with out config:  #11496